types/tar/index.d.ts

```diff
@@ -198,14 +198,22 @@ export const knownExtended: {
 export const headerSize: number;
 export const blockSize: number;
 
+/**
+ * A writable stream. Write tar data to it and it will emit entry events for each entry parsed from the tarball. This is used by tar.Extract.
+ */
+export interface Parse extends ParseStream {
+    on(event: 'end' | 'close', listener: () => void): this;
+    on(event: 'entry', listener: (entry: ReadEntry) => void): this;
+}
+
+export const Parse: {
+    new(opt?: { strict?: boolean, filter?: (path: string, entry: ReadEntry) => boolean, onentry?: (entry: ReadEntry) => void, onwarn?: (code: string, message: string, data: Buffer) => void }): Parse;
```

Comment:
I would have this be an interface like `ParseOptions` so that it can be easily referenced

Reply:
Added `ParseOptions` !